employee.rb

```diff
@@ -0,0 +1,60 @@
+require './reviews'
+
+class Employee
+
+  attr_reader :name, :salary, :reviews
+
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+  end
+
+  def add_review(text)
+    @reviews << text
+  end
+
+  def rate_satisfactory
+    @satisfactory = true
+  end
+
+  def rate_unsatisfactory
+    @satisfactory = false
+  end
+
+  def satisfactory?
+    return @satisfactory
+  end
+
+  def give_raise_by_percent(percent)
+    starting_salary = @salary
+    @salary = (percent + 1) * starting_salary
+  end
+
```

Comment:
Nice to have two methods for these two things.  Technically, though, you're not raising by a percent (50), you're raising by a fraction (.5).